docs/source/pillars/information_governance.md

```diff
@@ -2,137 +2,258 @@
-  -
+* - You must report on and share outcomes of each audit of your TRE organisation with the required bodies.
+  - This may include regulatory bodies or the organisations that manage accreditations you have
+  - Mandatory
+```
+
+**Supplier Management and Monitoring:** A structured approach to managing and monitoring relationships with external suppliers, vendors and contractors, including selection, contract management and compliance oversight.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-supplier-management-monitoring
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must ensure that suppliers, contractors and sub-contractors align with the security requirements of you TRE.
+  - These should be included as mandatory, non-functional requirements in during procurement and contracting.
+    This will also include contractor staff contracts for example, legal liability and NDAs.
+  - Mandatory
+* - You must monitor compliance of your suppliers with the terms of the contracts.
+  - This will include monitoring changes in the services and infrastructure being delivered and quality management within the contractor’s organisation.
+    This may be done through formal audit or by monitoring change and quality documentation provided by the supplier.
   - Mandatory
 ```
 
-### Compliance, monitoring and reporting
+**Asset Management Process:** A systematic approach to acquiring, operating, maintaining, and disposing of assets within an organization, aimed at maximizing their value and minimizing risks.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-asset-management-process
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must track and maintain physical assets if used to ensure security is maintained.
+  - Safe disposal of storage media and warranties maintained for critical infrastructure-maintained confidentiality, integrity and availability.
+  - Mandatory (where physical assets are in scope)
+```
 
-_The ability of the TRE organisation to monitor compliance with internal and external requirements, agreements, laws and standards._
+**Issue Management Process:** A systematic approach to identifying, tracking, resolving, and managing issues or problems that arise within a TRE organisation, aiming to minimize their impact and ensure timely resolution.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-compliance-monitoring-reporting
+:name: tab-ig-issue-management-process
+
 * - Statement
   - Guidance
   - Importance
-* - You must be able to audit your TRE organisation against whichever external standards are relevant to you.
-  - If you are publicly accredited against a standard, for example ISO27001, DSPT or CyberEssentials+, you must have processes in place to ensure you remain compliant
+* - You must log, track and resolves issues resulting from deviations from processes, incidents and audit findings.
+  - This process should be tracked through an electronic record and workflow system with records retained.
```

Comment:
Why is it mandatory to do this electronically? A comprehensive paper system would be more useful than a cursory "stick everything in a folder on a shared drive" electronic one.

Reply:
I guess there's some ambiguity in how we're writing the spec. The `Statement` is mandatory, but arguably `Guidance` is always advisory?

Reply:
```suggestion
  - This process could, for example, be tracked through an electronic record and workflow system with records retained.
```